jina/clients/base/http.py

```diff
@@ -1,5 +1,6 @@
 import asyncio
 from contextlib import nullcontext, AsyncExitStack
+from sgqlc.endpoint.http import HTTPEndpoint as SgqlcHTTPEndpoint
```

Comment:
This import needs to be lazy, we are not sure this will be installed in every client.